src/download.js

```diff
@@ -35,15 +33,9 @@ function fetchNodeSource (path, url, log) {
         log.info('Complete...')
       }
     }).on('error', reject)
-      .pipe(unZip().on('error', reject))
-      .pipe(unTar(path, {
-        map (header) {
-          header.name = header.name.replace(prefix, '')
-          return header
-        }
-      }))
+      .pipe(unTar({ x: 1, strip: 1, cwd }))
       .on('error', reject)
-      .on('end', () => resolve(log.info('Extracted to: ' + path)))
+      .on('close', () => resolve(log.info('Extracted to: ' + cwd)))
```

Comment:
Something is still broken here. If download is run it exists immediately, and this isn't logged either.